### PR TITLE
Add coloring of statusbar.current tab in view

### PR DESCRIPTION
**Pull request by quite**
This theme color applies to the tab title text of the statusbar tab that
is currently shown.

The (somewhat confusingly named) `statusbar.active` theme color now
applies to all other tabs (before, it applied to all tabs).

Coloring of a tab that is highlighted/has new messages is done as before
using the `statusbar.new` theme color.

The default color is set to `green`.

### CODE REVIEW

**Comment by quite** [comment]
I did not add this item to all the current themes...

File: src/config/theme.c [hunk 1]
```diff
@@ -99,6 +99,7 @@ theme_init(const char *const theme_name)
     g_hash_table_insert(defaults, strdup("statusbar.text"),          strdup("white"));
     g_hash_table_insert(defaults, strdup("statusbar.brackets"),      strdup("cyan"));
     g_hash_table_insert(defaults, strdup("statusbar.active"),        strdup("cyan"));
+    g_hash_table_insert(defaults, strdup("statusbar.current"),       strdup("green"));
```
Comment (jubalh) [inline src/config/theme.c]: please use cyan here, so that it behaves like before for users.

File: src/ui/statusbar.c [hunk 2]
```diff
@@ -337,6 +337,9 @@ _status_bar_draw_extended_tabs(int pos)
         pos = _status_bar_draw_bracket(is_current, pos, "[");
 
         int status_attrs = theme_attrs(THEME_STATUS_ACTIVE);
```
Comment (jubalh) [inline src/ui/statusbar.c]: Maybe do:
int status_attrs;
and then use if/else.

**Comment by jubalh** [comment]
First of all: thanks for creating an issue and then also the pull request with the change!
Generally I looks very good. I also like how you documented everything and how you wrote the commit message. Thanks for this!

File: src/ui/statusbar.c (hunk 2 above)
Comment (quite) [reply src/ui/statusbar.c]: I guess then I should not change *from* that variant in the other place, as I did?

Comment (quite) [reply src/ui/statusbar.c]: I'll change any way you prefer. I just did it the way it's more readable for me, starting generic, going towards specific.

Comment (jubalh) [reply src/ui/statusbar.c]: It would just be nice not to assign a value twice if we can go around it.

**Comment by quite** [comment]
Done! Don't forget profanity-im/profanity-im.github.io#35

**Comment by jubalh** [comment]
Thanks for your contribution!

**Pull request merged**